recirq/quantum_chess/quantum_board.py

```diff
@@ -93,14 +94,15 @@ def __init__(
         ] = enums.ErrorMitigation.Nothing,
         noise_mitigation: Optional[float] = 0.0,
         transformer: Optional[ct.CircuitTransformer] = None,
+        reset_starting_states=True,
     ):
         self.device = device
         self.sampler = sampler
         if device is not None:
             self.transformer = (
                 transformer or ct.ConnectivityHeuristicCircuitTransformer(device)
             )
-        self.with_state(init_basis_state)
+        self.with_state(init_basis_state, True)
```

Comment:
True is the default and is nit strictly needed.  Do you think keeping "True" makes things more clear?

Reply:
Ah, I think that's left over from a time when true might not have been the default. It's probably not necessary to keep.